src/main/webapp/search_area.js

```diff
@@ -90,6 +91,10 @@ class SearchArea {
     this.filterInputArea.appendChild(this.filterDataList);
     this.organizationSearchArea.appendChild(this.filterInputArea);
 
+    this.activeFilterArea = document.createElement("div");
+    this.activeFilterArea.setAttribute("class", "filter-holder"); //REMOVE THIS LINE
```

Comment:
Why do you have this remove this line comment?

Reply:
forgot to remove that, actually do want that line:)